board.cpp

```diff
@@ -38,27 +38,27 @@ int Board::count(const int sphereType, const int gemType) const {
   return count;
 }
 
-bool Board::isMoveValid(const std::vector<std::tuple<Element, Element, Element>> moves) const {
+bool Board::isMoveValid(const std::vector<Move> moves) const {
 
-  MoveMap moveMap(moves);
-  for (auto element : ELEMENTLIST) {
-    for (auto element2 : ELEMENTLIST) {
-      if (moveMap.inputCount(element, element2) > count(element, element2)) {
-        return false;
-      }
-    }
-  }
+   MoveMap moveMap(moves);
+   for (auto element : ELEMENTLIST) {
+     for (auto element2 : ELEMENTLIST) {
+       if (moveMap.inputCount(element, element2) > count(element, element2)) {
+         return false;
+       }
+     }
+   }
```

Comment:
Remove excess spaces.
```suggestion
  MoveMap moveMap(moves);
  for (auto element : ELEMENTLIST) {
    for (auto element2 : ELEMENTLIST) {
      if (moveMap.inputCount(element, element2) > count(element, element2)) {
        return false;
      }
    }
  }
```